jwst/wfs_combine/wfs_combine.py

```diff
@@ -315,42 +314,36 @@ def create_combined(self, im_1_a, im_2_a):
             combined ERR array
         """
 
-        sci_data_1 = im_1_a.data.astype(np.float64)
-        sci_data_2 = im_2_a.data.astype(np.float64)
-
-        dq_data_1 = im_1_a.dq.copy()
-        dq_data_2 = im_2_a.dq.copy()
-        err_data_1 = im_1_a.err.copy()
-        err_data_2 = im_2_a.err.copy()
-
-        bad1 = np.bitwise_and(dq_data_1, dqflags.pixel['DO_NOT_USE'])
```

Comment:
The bug was here (and 2 lines down).  `np.bitwise_and` returns an array of 0 and 1 values, and cannot be used to index another array without being converted to a boolean array.  Once converted, it can have the True/False values flipped via that `~` operator.